prow/cluster/jobs/rh-ibm-synergy/multicloud-operators-cluster-controller/rh-ibm-synergy.multicloud-operators-cluster-controller.master.yaml

```diff
@@ -2,6 +2,14 @@ presets:
 - env:
   - name: GOPRIVATE
     value: github.ibm.com,github.com/rh-ibm-synergy
+- volumes:
+  - name: ssh-keys-ssh-secret
+    secret:
+      secretName: ssh-secret
```

Comment:
Looks good, although the secret volume is already created, because we add 
```
    decoration_config:
      ssh_key_secrets:
      - ssh-secret
```